.env.example

```diff
@@ -1,33 +1,36 @@
-# Url for Mudita Center API
-VITE_MUDITA_CENTER_SERVER_URL=
-
-# Whether to allow updates in development mode
-DEV_APPUPDATE_ENABLED=
+# [CI Mode] Mocks behavior of e2e tests run in CI. Default "false"; set "true" to enable.
+E2ECI=
 
 # [Setup Mode] URL for mudita-dev-resources repo where gt-pressura font is placed
 FONTS_DIRECTORY_URL=
 
 # [Setup Mode] Access token for dummy-account that has an access to Mudita private repositories
-GITHUB_ACCESS_TOKEN=
+VITE_GITHUB_ACCESS_TOKEN=
```

Comment:
This is a no-go for me. According to https://vite.dev/guide/env-and-mode.html#env-variables this env may be included into built app.

> To prevent accidentally leaking env variables to the client, only variables prefixed with VITE_ are exposed to your Vite-processed code.